### ISSUE TITLE
AWS amplify Cognito User Pool MFA Issue on Mobile phone

### ISSUE THREAD

**Issue by chikason** [aws-amplify/amplify-cli#2149]
![n1](https://user-images.githubusercontent.com/52418760/63750120-c4980680-c8a4-11e9-8445-ea20f9e5d268.PNG)
![n2](https://user-images.githubusercontent.com/52418760/63750135-cb267e00-c8a4-11e9-8df5-9cc1918ea518.PNG)

** I am not receiving MFA code authentication on my mobile phone for my user signUp authentication and verification**

** AWS Amplify **

** Provide additional details e.g. code snippets **

**Comment by chikason** [comment]
@kaustavghosh06 yes I did it through CLI. 
I have had one that worked but can't remember if their is extra that I should do. 
But from the MFA page it pointed out that for you to use sns you have give the permission by using "Create role"  button on the page but their is no such button on  MFA section.

**Comment by chikason** [comment]
@yuth  I did it myself, through manual configuration and I chosed SMS message for MFA types.

NOTE: from the snapshot I uploaded above
1. just bellow "SMS Text Message" check box it says  in order for that to work I must request a spending limit increase for SNS. 
How do I do that? 
2. Just before the "new role name" at the bottom, It says amazon cognito needs your permission to send SMS message to your user. 
How do I provide this permission? 
Is the permission difference from the MFA configuration that I manually closed. 
Thanks.
